backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/CalendarsService.cs

```diff
@@ -222,5 +225,49 @@ private async Task AddMeetingsFromCalendar(string email, IEnumerable<CalendarVis
                 await _meetingService.AddGoogleCalendarMeetings(item.TeamId, events, userId);
             }
         }
+        private async Task AddMeetingsToCalendar(long? teamId)
+        {
+            var meetings = await _context.Meetings.Where(x => x.TeamId == teamId).ToListAsync();
+
```

Comment:
I don't think we have to enumerate meetings and check for any

Reply:
I believe we can do this in this situation since we use them later

Reply:
Ok